tools/platyPS/GenerateHelp.ps1

```diff
@@ -0,0 +1,133 @@
+﻿[CmdletBinding()]
```

Comment:
Let's make this a full-fledged module, and add online help for it.  You can see an example of what this looks like here: https://github.com/markcowl/azure-powershell/tree/bootstrap/tools/AzureRM.BootStrapper  This is a script module with help created by PlatyPS